.travis.yml

```diff
@@ -1,50 +1,100 @@
+# Variables needed for this script are:
+
+# Case 1: Stored on Travis Settings
+  # For all environments:
+    # - DOCKER_CI_REPO_NAME
+    # - DOCKER_USERNAME
+    # - DOCKER_PASSWORD
+
+  #For dev environment:
+    # - AWS_ACCOUNT_USER_ID_DEV
+    # - AWS_ACCOUNT_SECRET_DEV
+    # - AWS_PROFILE_DEV <-- is this needed?
+
+  #For Prod environment:
+    # - AWS_ACCOUNT_USER_ID_PROD
+    # - AWS_ACCOUNT_SECRET_PROD
+    # - AWS_PROFILE_PROD <-- is this needed?
+
+  #For Demo environment:
+    # - AWS_ACCOUNT_USER_ID_DEMO
+    # - AWS_ACCOUNT_SECRET_DEMO
+    # - AWS_PROFILE_DEMO <-- is this needed?
+
+# Case 2: Created when Travis CI runs
+    # - IMAGE_TAG
+
 sudo: required
 
-language: node_js
-node_js:
-  - lts/*
+# language: node_js
+# node_js:
+#   - lts/*
+language: generic
+services:
+  - docker
+
+env:
+  global:
+    - COMMIT=${TRAVIS_COMMIT::7}
+    #Add IAMGE_TAG variable to build docker image
+    - IMAGE_TAG=${TRAVIS_COMMIT::7}
 
 before_install:
   # Removed command as pyenv is no longer found during deploy
   #- pyenv global 2.7.14  # For removing SNIMissingWarning warnings
-  - curl https://install.meteor.com | /bin/sh
-  - export PATH="$HOME/.meteor:$PATH"
+  # - curl https://install.meteor.com | /bin/sh
+  # - export PATH="$HOME/.meteor:$PATH"
   - pip install --user awscli
   - export PATH=$PATH:$HOME/.local/bin
   - curl -o $HOME/.local/bin/ecs-cli
     https://s3.amazonaws.com/amazon-ecs-cli/ecs-cli-linux-amd64-latest &&
     chmod +x $HOME/.local/bin/ecs-cli
 
-cache:
-  directories:
-    - node_modules
+# cache:
+#   directories:
+#     - node_modules
 
 install:
-  - meteor npm install
+  # - meteor npm install
+  # #Adding script build docker image
+  - docker build -t $DOCKER_CI_REPO_NAME . 
+
+# script:
+#   - npm t
 
-script:
-  - npm t
+# #Adding script deploy docker image to docker repository
+
+after_success:
+  - docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
+  - docker build -t $DOCKER_CI_REPO_NAME:latest .
+  - docker tag $DOCKER_CI_REPO_NAME:latest $DOCKER_CI_REPO_NAME:$IMAGE_TAG
+  - docker push $DOCKER_CI_REPO_NAME:latest
+  - docker push $DOCKER_CI_REPO_NAME:$IMAGE_TAG
 
 matrix:
   include:
-    - env: DEPLOY=none
+    # add deploy stage for dev
+    - env: DEPLOY=dev
+      if: branch = master
+      deploy:
+        provider: script
+        skip_cleanup: true
+        script: PROFILE=$AWS_PROFILE_DEV AWS_ACCESS_KEY_ID=$AWS_ACCOUNT_USER_ID_DEV AWS_SECRET_ACCESS_KEY=$AWS_ACCOUNT_SECRET_DEV AWS_REGION=$AWS_DEFAULT_REGION ./deploy.sh -d
+        on:
+          branch: master
     - env: DEPLOY=demo
       if: tag IS present
       deploy:
         provider: script
         skip_cleanup: true
```

Comment:
`skip_cleanup: true` is deprecated in Travis CI: we can safely remove this line:
See [Travis View confif comment for more details](https://travis-ci.org/Unee-T-INS/frontend/jobs/636699944/config#rccb_frontend:.travis.yml@2bca01be674570b8.27)

Reply:
fix in commit: https://github.com/Unee-T-INS/frontend/pull/7/commits/bc2335722d03b53820aee8c35b28f2aff72aab77